src/coupled/AM3/control/cable/CM3/cbl_cbm_mod.F90

```diff
@@ -110,8 +110,9 @@ SUBROUTINE cbm_expl( mp, nrb, ktau,dels, air, bgc, canopy, met,                &
                    ) !reducedLAIdue2snow 
 
 !Ticket 331 refactored albedo code for JAC
-CALL snow_aging(ssnow%snage,mp,dels,ssnow%snowd,ssnow%osnowd,ssnow%tggsn(:,1), &
-                ssnow%tgg(:,1),ssnow%isflag,veg%iveg,soil%isoilm) 
+!#539 - move snow aging. 
+!CALL snow_aging(ssnow%snage,mp,dels,ssnow%snowd,ssnow%osnowd,ssnow%tggsn(:,1), &
+!                ssnow%tgg(:,1),ssnow%isflag,veg%iveg,soil%isoilm) 
 
```

Comment:
Why are we keeping the lines commented out? Nobody is going to come back to it and clean up after. Can we remove the lines before merging in please?

Otherwise I'm happy for this to go. Don't worry about the CM2 changes, that directory will be removed with the shuffling of files.

Reply:
done - see 8ab0153